### ISSUE TITLE
Add support for `Include`

### ISSUE THREAD

**Issue by ghost** [deanishe/alfred-ssh#11]
The `~/.ssh/config` file supports an `Include` directive to load more config files, making it easier to separate groups of hosts. For example:

```
# ~/.ssh/config
IdentityFile ~/.ssh/id_rsa
IdentityFile ~/.ssh/id_rsa_work

Include ~/.ssh/config_work
Include ~/.ssh/config_private
```

Hosts from these `Include`d configuration files should also be shown.

**Comment by deanishe** [comment]
Duplicate of #3, which was attempted in #7.

The workflow uses a 3rd-party library to parse ssh config files. If you can find one that supports `Include`, I can integrate it.

**Comment by ghost** [comment]
Looks like https://github.com/danieljimenez/ssh_config should have supported that, but it doesn't 🤔 Seems he misunderstood how `Include` works. I'll see if I can get that working.

**Comment by deanishe** [comment]
Yeah, it doesn’t work like it’s supposed to.

Now you bring it up, i should probably revert to the old library with no `Include` support, rather than one with broken support.

`Include` support would be awesome. I had another look through the available libs, but none I could find supports it :(

**Comment by alex-mohemian** [comment]
@deanishe I'll fix it in the coming weeks. My `go` skills suck but that looks incredibly easy to implement.

**Comment by georgesnow** [comment]
@deanishe I know this is old issue. but I will take a look and see if can figure out recursive ssh configs. doesn't look like anyone else has attempted to figure it out.

**Comment by deanishe** [comment]
Oops. [I have a working parser here](https://git.deanishe.net/deanishe/sshconf). I'm afraid I just haven't bothered to tidy it up and integrate it yet.

**Comment by ghost** [comment]
Looks like I never got around to that either 🙈 

@deanishe would be cool if you could include that!

**Comment by ideologysec** [comment]
Any updates on this? I saw a mention elsewhere on your Alfred-Workflows issues (awesome lib btw, thanks very much!) that you're rewriting most things in Python3 - is this Include file support getting deferred until the automation landscape settles down a bit?

**Comment by deanishe** [comment]
> is this Include file support getting deferred until the automation landscape settles down a bit?

No, I'm just really busy with work atm. It's not helping that when I look at the workflow's code, I want to rewrite the whole thing…

**Comment by zimmortal** [comment]
I've solved it. You can check it  https://github.com/zimmortal/alfred-ssh
